app/src/main/java/com/cdio/solitaire/fragment/CameraFragment.kt

```diff
@@ -73,12 +73,6 @@ class CameraFragment : Fragment(), SensorEventListener {
         _fragmentCameraBinding = null
         super.onDestroyView()
 
-        // Exit fullscreen mode
-        activity?.let {
-            it.requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_UNSPECIFIED
-            it.window.clearFlags(WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS)
```

Comment:
The fullscreen part shouldn't be removed after all. Just insert
`requireActivity().window.clearFlags(WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS)`